integration-test/pom.xml

```diff
@@ -64,6 +90,33 @@
 
     <dependency>
       <groupId>${project.groupId}</groupId>
+      <artifactId>livy-integration-test-minicluster-dependencies_2.10</artifactId>
```

Comment:
If we shift to Scala 2.11 for default supported version, how we can change to use 2.11 one.